docs/core_modules/data_modules/documents_and_nodes/usage_nodes.md

```diff
@@ -32,4 +32,21 @@ The `RelatedNodeInfo` class can also store additional `metadata` if needed:
 
 ```python
 node2.relationships[NodeRelationship.PARENT] = RelatedNodeInfo(node_id=node1.node_id, metadata={"key": "val"})
-```
\ No newline at end of file
+```
+
+### Customizing the ID
+
+Each node has an `node_id` property that is automatically generated if not manually specified. This ID can be used for 
+a variety of purposes; this includes being able to update nodes in storage, being able to define relationships
+between nodes (through `IndexNode`), and more.
+
+
+You can also get and set the `node_id` of any `TextNode` directly (and also `Document` objects as well)!
```

Comment:
very nit: I find "and also `Document` objects as well" to be slightly confusing. 

Previously users had a lot of confusion about node vs. doc and `node_id` vs. `doc_id`. Let's not maybe not advertise the `node_id` on Document?